modules/core/arrow-extras-data/src/main/kotlin/arrow/data/fingertree/internal/Affix.kt

```diff
@@ -0,0 +1,35 @@
+package arrow.data.fingertree.internal
+
+internal sealed class Affix<A> {
```

Comment:
You can use @AutoFold to generate a `fold` function to use instead of the `when` below.